edc-controlplane/edc-controlplane-postgresql-hashicorp-vault/src/main/docker/Dockerfile

```diff
@@ -10,13 +10,14 @@
 #  Contributors:
 #       Mercedes-Benz Tech Innovation GmbH - Initial Dockerfile
 #
-FROM alpine:3.16.2 as otel
+
+FROM alpine:3.17.1 as otel
 
 ENV OTEL_AGENT_LOCATION "https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.1/opentelemetry-javaagent.jar"
 
 RUN wget ${OTEL_AGENT_LOCATION} -O /tmp/opentelemetry-javaagent.jar
 
-FROM alpine:3.16.2
+FROM alpine:3.17.1
```

Comment:
For Java based applications please use [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin) images as base image ([TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-2)).